src/wmnds/patterns/question/summary/_summary.njk

```diff
@@ -14,55 +14,118 @@
     {{
         wmndsQuestion({
             text: question
-        }) | trim | indent(4)
+        })
     }}
 
-    {%- if tables -%}
-        {%- for table in tables %}
-            {{ table | trim | indent(4) | safe }}
-        {%- endfor %}
+    {% if tables %}
+        {% for table in tables %}
+            {{ table | safe }}
+        {% endfor %}
     {% else %}
     {{
         [
             wmndsTable({
-                rows: 3,
-                cols: 3,
-                cellClasses: ["", "", "wmnds-text-align-right"],
                 title: "Personal Details",
-                noHeaders: true,
-                classNames: "wmnds-m-b-xl",
-                values: [
-                    ["Name", "Andy Street", "<a href=\"#\" title=\"link title\" target=\"_self'\" class=\"wmnds-link\">Change</a>"],
-                    ["Date of Birth", "1 January 2020", "<a href=\"#\" title=\"link title\" target=\"_self'\" class=\"wmnds-link\">Change</a>"],
-                    ["Contact Details", "07700 900 457<br>andy.street@example.com", "<a href=\"#\" title=\"link title\" target=\"_self'\" class=\"wmnds-link\">Change</a>"]
-                ]
-            }) | trim | indent(4),
+                hideHeader: true,
+                classes: "wmnds-m-b-xl"
+            }),
 
             wmndsTable({
-                rows: 3,
-                cols: 3,
-                cellClasses: ["", "", "wmnds-text-align-right"],
                 title: "Ticket Details",
-                noHeaders: true,
-                classNames: "wmnds-m-b-xl"
-            }) | trim | indent(4),
+                hideHeader: true,
+                tableRows: [
+                    [
+                        {
+                            contentText: "Cell"
+                        },
+                        {
+                            contentText: "Cell"
+                        },
+                        {
+                            contentText: "Cell",
+                            classes: "wmnds-text-align-right"
+                        }
+                    ],
+                    [
+                        {
+                            contentText: "Cell"
+                        },
+                        {
+                            contentText: "Cell"
+                        },
+                        {
+                            contentText: "Cell",
```

Comment:
Can you make the last cell a button as link with the text "Change" like how it would be in a live form.

Do this for all last cells.